roles/scenario7/tasks/fix_podified.yml

```
@@ -0,0 +1,43 @@
+---
+- name: find the floating ip
+  ansible.builtin.shell: |
+    {{ osp_get_credentials_cmd }}
+
+    {{ oc_osp_cmd }} openstack port list --server scenario-{{ scenario }}-vm -c "Fixed IP Addresses" -f json | jq -r '.[0]["Fixed IP Addresses"][0].ip_address'
+  delegate_to: 127.0.0.1
+  register: floating_ip
+
+- name: Find OVN POD name
+  block:
+  - name: Get OVN PODs
+    ansible.builtin.shell: "{{ oc_bin }} -n {{ oc_namespace }} get pods -l service=ovsdbserver-nb -o name | head -n 1"
+    register: ovn_pods
+    delegate_to: 127.0.0.1
```

should this fail early if stdout is empty?